src/main/java/zero/task/Event.java

```diff
@@ -0,0 +1,30 @@
+package zero.task;
+
+import java.time.LocalDateTime;
+
+/**
+ * The {@code Event} class represents a task that occurs within a specific time frame.
+ * It is a subclass of {@code Task} and adds two {@code LocalDateTime} fields to store the start (from)
+ * and end (to) times.
+ */
+public class Event extends Task {
+    protected LocalDateTime from;
```

Comment:
Similarly over here the fact that it's a date could be added e.g. fromDate. Other such constants in other classes should also be corrected!